index.html

```diff
@@ -224,27 +230,46 @@ <h4 id="about-intro">About me</h4>
           </div>
         </div>
       </section>
-      <!-- contact setion -->
-      <section id="contact-sections">
+      <!-- contact section -->
+      <footer id="contact-sections">
         <div id="contact-intro">
           <h4>
             I'm always interested in hearing about new projects, so if you'd
             like to chat please get in touch.
           </h4>
         </div>
-        <form action="action" id="personal-info">
-          <input type="text" placeholder="Full name" id="fname" name="fname" />
+        <form
+          action="https://formspree.io/f/personal-info"
```

Comment:
- [ ] please it would be better to add you id to submit a from right now when I submit it doesn't submit please try to fix that 
![image](https://user-images.githubusercontent.com/61790587/171293131-86f57435-1046-4521-9c57-8eba48252cea.png)